.github/workflows/ci-app-tests.yml

```diff
@@ -11,7 +11,7 @@ on:
       - ".github/workflows/ci-app-tests.yml"
       - "src/lightning_app/**"
       - "tests/tests_app/**"
-      - "examples/app_*"  # some tests_app tests call examples files
+      - "examples/app_*/**"  # some tests_app tests call examples files
```

Comment:
This now matches what's in https://github.com/Lightning-AI/lightning/blob/609b258fe80dd09adb30f9a37455a88e1bba3732/.github/checkgroup.yml#L226. The job should run now, as the bot expects